CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ in the naming of release branches.
 
 ## [Unreleased]
 
+
+### Changed
+
+- Moved Cardano.Ledger.Shelley.Orphans to Cardano.Ledger.Orphans  in the core modules
+- Moved Cardano.Ledger.Shelley.PoolParms to Cardano.Ledger.PoolParams  in the core modules
+- Moved Cardano.Ledger.Shelley.EpochBoundary to Cardano.Ledger.EpochBoundary  in the core modules
+- Added deprecated modules that import the moved ones and export their contents
```

Comment:
We should move these to the top of the `Changed` section below.

Reply:
done